packages/api-fetch/src/index.native.js

```diff
@@ -0,0 +1,10 @@
+
+function apiFetch( options ) {
+	// eslint-disable-next-line no-console
+	console.warn( 'apiFetch called with options', options );
```

Comment:
Instead of `console.warn`, could we make this `console.info` or `console.log` so we don't add another yellow box warning to the debug apps?

Reply:
I think it's pretty alarming if the app is trying to make an api request and we should be aware of it. At least it's as alarming as state update warning or deprecation notice imo.
What we could do if you think those warnings are disrupting the experience when testing on mobile is disable them completely. If we want to see those we can simply attach a debugger.

Reply:
Disabled in https://github.com/wordpress-mobile/gutenberg-mobile/pull/1366/commits/fb6db571f796adef5eeaaa310e3d0c4ec88b1028

Reply:
> I think it's pretty alarming if the app is trying to make an api request and we should be aware of it.

I'm seeing that warning every time I run the app. Is that expected?

Reply:
Yeah I see it as well, I haven't tracked down the issue yet, but we might have to when we tackle editing image options since it seems related to media.

Reply:
Thanks for explaining. If this is a we-need-to-fix-this-as-soon-as-reasonably-possible type of thing, I have no concern about leaving the warning if you want. Alternatively, removing the warning and filing an issue to add the warning back in (and fix whatever is causing it to fire) could make sense so this doesn't get forgotten.